pystac/extensions/eo.py

```diff
@@ -545,7 +545,7 @@ def migrate(
                         obj["stac_extensions"] = []
                     if view.SCHEMA_URI not in obj["stac_extensions"]:
                         obj["stac_extensions"].append(view.SCHEMA_URI)
-                    if not "view:{}".format(field) in obj["properties"]:
```

Comment:
should this behavior change be in the release PR?

Reply:
Don't believe this is a behavior change, just a move of `not` to be `not in` (recommended by ruff)...I think that's just a lint?

Reply:
I didn't look at it enough to see it was the same, but why's that only showing up now?

Reply:
I ran `ruff . --fix` to see if it picked something up. I thought it was better to fold into an existing PR rather than make it Its Own Thing.

Maybe I should create a PR to add ruff instead?

Reply:
I think it should be it's own commit b/c this is the release pr.

If we're going to use ruff (which I am 👍🏻 on), it should be part of pre-commit and the ci build.

Reply:
It is it's own commit? cf6c17e222f1faf84fc0cfe7b1020f9413c6d9c5

> If we're going to use ruff (which I am 👍🏻 on), it should be part of pre-commit and the ci build.

https://github.com/stac-utils/pystac/issues/1030

Reply:
I meant PR. But whatever, :shipit: